index.html

```diff
@@ -159,141 +156,401 @@ <h2>
+                    [=this=]'s [=Node/node document=].
+                    </li>
+                    <li>[=Reject=] |promise| with a
+                    {{"WrongDocumentError"}} {{DOMException}}.
+                    </li>
+                  </ol>
+                </li>
+                <li>Return <var>promise</var>.
+                </li>
+              </ol>
+            </li>
+            <li>If the user agent's [=pointer-lock target=] is an element whose
+            [=shadow-including root=] is not equal to [=this=]'s
+            [=shadow-including root=], then:
+              <ol>
+                <li>[=Queue an element task=] on the [=user interaction task
+                source=], given this, to perform the following steps:
+                  <ol>
+                    <li>[=Fire an event=] named {{pointerlockerror}} at
+                    [=this=]'s [=Node/node document=].
+                    </li>
+                    <li>[=Reject=] <var>promise</var> with a
+                    "{{InvalidStateError}}" {{DOMException}}.
+                    </li>
+                  </ol>
+                </li>
+                <li>Return <var>promise</var>.
+                </li>
+              </ol>
+            </li>
+            <li>If the user agent's [=pointer-lock target=]'s
+            [=shadow-including root=] is equal to [=this=]'s [=shadow-including
+            root=], then:
+              <ol>
+                <li>If [=options=] are equivalent to the current [=pointer-lock
+                options=]
+                  <ol>
+                    <li>Set [=pointer-lock target=] to [=this=]
+                    </li>
+                    <li>[=Queue an element task=] on the [=user interaction
+                    task source=], given this, to perform the following steps:
+                      <ol>
+                        <li>[=Fire an event=] named {{pointerlockchange}} at
+                        [=this=]'s [=Node/node document=].
+                        </li>
+                        <li>[=Resolve=] the <var>promise</var>.
+                        </li>
+                      </ol>
+                    </li>
+                  </ol>
+                </li>
+                <li>If [=options=] are not equivalent to the user agent's
+                current [=pointer-lock options=] and [=options=] are not
+                supported:
+                  <ol>
+                    <li>[=Queue an element task=] on the [=user interaction
+                    task source=], given this, to perform the following steps:
+                      <ol type='i'>
+                        <li>[=Reject=] <var>promise</var> with a
+                        "{{NotSupportedError}}" {{DOMException}}.
+                        </li>
+                      </ol>
+                    </li>
+                    <li>Return <var>promise</var>.
+                    </li>
+                  </ol>
+                </li>
```

Comment:
We don't think this makes any sense... there is only 1 option and unsupported WebIDL things are dropped on the floor. 

```suggestion
```